orchagent/saihelper.cpp

```diff
@@ -454,15 +443,14 @@ sai_status_t initSaiPhyApi(swss::gearbox_phy_t *phy)
 
     attr.id = SAI_SWITCH_ATTR_FIRMWARE_MAJOR_VERSION;
     status = sai_switch_api->get_switch_attribute(phyOid, 1, &attr);
-    if (status != SAI_STATUS_SUCCESS)
-    {
+    if (status == SAI_STATUS_SUCCESS) {
+       phy->firmware_major_version = string(attr.value.chardata);
+    } else if( status == SAI_STATUS_NOT_SUPPORTED ) {
+       phy->firmware_major_version = "";
```

Comment:
I am not sure the empty string is good enough here. Should it be "N/A" or "Not Supported"?

Reply:
I'm fine with changing it to N/A. I don't think there are any strict requirements here.